src/cui.cpp

```diff
@@ -199,6 +200,11 @@ void Line::log() {
             << "\t" << recv_value << std::endl;
 }
 
+void Line::log_json() {
+  printf("{\"name\":\"%s\",\"PID\":%d,\"UID\":%d,\"RX\":%f,\"TX\":%f }",
+        m_name, m_pid, m_uid, recv_value, sent_value);
```

Comment:
Sorry for my ignorance: do you mean 'display mode' as in catering to the type of terminal they may be viewing it in, or adding multiple display modes, like optional multi-line json, special character terminated, etc?